clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -83,9 +83,6 @@ void visualstudio::Linker::constructMSVCLibCommand(Compilation &C,
     }
     CmdArgs.push_back(II.getFilename());
   }
-  if (Args.hasArg(options::OPT_fsycl_link_EQ) &&
```

Comment:
Do we need this?
https://github.com/intel/llvm/blob/fb70c93770981a9ea1ae74d8541de56a150078c9/clang/lib/Driver/ToolChains/MSVC.cpp#L68

Reply:
Nope - needs to be cleaned out as well.